static_src/stores/route_store.js

```diff
@@ -76,6 +76,8 @@ class RouteStore extends BaseStore {
         cfApi.createRoute(domainGuid, spaceGuid, host, path).then((res) => {
           const routeGuid = res.metadata.guid;
           cfApi.putAppRouteAssociation(appGuid, routeGuid);
+        }).catch(() => {
+          // Do nothing
```

Comment:
I found that if I didn't put that catch there then it would continue in the then callback which would break things. I'll just say actual error handling is done in the api code.